src/Engine/ProtoCore/DSASM/Executive.cs

```diff
@@ -705,7 +705,7 @@ private bool GetSymbolIndexedIntoList(string symbol, out List<int> indexList)
                 svThisPtr.metaData.type != Constants.kInvalidIndex)
             {
                 int runtimeClassIndex = svThisPtr.metaData.type;
-                ClassNode runtimeClass = core.ClassTable.ClassNodes[runtimeClassIndex];
+                ClassNode runtimeClass = exe.classTable.ClassNodes[runtimeClassIndex];
```

Comment:
Its small caps, but yea ill change it to capital if it ends up being accessed directly.
Will address in a succeeding PR. thanks